.env.example

```diff
@@ -2,6 +2,5 @@
 ANS_HANDLE=your_ans_handle
 SPECTRAL_API_KEY=your_spectral_api_key
 OPENAI_API_KEY=your_openai_api_key
-SPECTRAL_API_AGENTS_ENDPOINT=spectral_company_endpoint
```

Comment:
Wouldn't be better to comment it out or set it to an empty value and then make sure if it's empty it uses the default?

Reply:
Or actually really easily set this to `https://agents-api.spectrallabs.xyz"?

Reply:
I'm actually inclined to remove it. This is expected to only differ when used internally to test in other environments, and in such cases the user can manually change it. I don't like the idea of commenting it out as it could cause confusion when copying `.env.example` into `.env`